src/pages/index.js

```diff
@@ -43,6 +49,37 @@ const IndexPage = ({ data }) => {
           </MissionDescription>
         </MissionSection>
       </HeroContainer>
+
+      <ByTheNumbers>
+        <ByTheNumbersHeader>{homepageQuery.numbers_header}</ByTheNumbersHeader>
+
+        <Numbers>
```

Comment:
can you render these using a loop for cleaner code since they're all the same structure? also not sure if you need that extra div that you have as a child of each `Number`